include/revng/Model/schema.yml

```diff
@@ -0,0 +1,701 @@
+preamble: |
+  #pragma once
+
+  #include <compare>
+  #include <optional>
+
+  #include "llvm/ADT/StringRef.h"
+  #include "llvm/Support/YAMLTraits.h"
+  #include "revng/ADT/RecursiveCoroutine.h"
+  #include "revng/Model/Identifier.h"
+  #include "revng/Model/TupleTree.h"
+  #include "revng/Model/VerifyHelper.h"
+  #include "revng/Support/MetaAddress.h"
+
+types:
+  # Enums
+  - name: ABI
+    type: enum
+    members:
+      - name: SystemV_x86_64
```

Comment:
Do enums have something else except the name?
If not, make it a list of strings.
I guess they `doc`.

Reply:
Yes, enum members can have a `doc` (see `FunctionEdgeType` for instance)